src/packet.c

```diff
@@ -133,7 +133,7 @@ void PacketReinit(Packet *p)
             PacketAlertRecycle(p->alerts.alerts, p->alerts.cnt);
         p->alerts.cnt = 0;
     }
-    p->pcap_cnt = 0;
+    PcapPacketCntReset(p);
```

Comment:
here we turn the goal of a small structure shrink into a per packet runtime overhead with a fairly complex check. I don't think this is acceptable